### PR TITLE
CL-457 cura connect visual

### PR DESCRIPTION
**Pull request by sedwards2009**
Jira CL-457

This PR contains work for the following subtasks:

* CL-470 Add icons to the printer state box on the monitor tab.
* CL-474 Fix the image used for the camera icon
* CL-473 Print job duration time has the wrong format. Use the Cura format from the Prepare tab.
* CL-476 "View print jobs" and "View printers" buttons should be blue
* CL-469 Remove the "Printer Group" text from the sidebar.
* CL-468 The sidebar in the monitor tab should have a grey background

### CODE REVIEW

File: plugins/UM3NetworkPrinting/PrinterInfoBlock.qml [hunk 1]
```diff
@@ -143,15 +136,28 @@ Rectangle
...
+                    width: 40 * screenScaleFactor
+                    height: width
+                    anchors.right: printProgressArea.left
+                    anchors.rightMargin: UM.Theme.getSize("default_margin").width
                     source: "camera-icon.svg"
-                    width: sourceSize.width
-                    height: sourceSize.height * width / sourceSize.width
-                    color: "white"
                 }
+
```
Comment (nallath) [inline plugins/UM3NetworkPrinting/PrinterInfoBlock.qml]: Remove commented out code.

File: resources/qml/Cura.qml [hunk 2]
```diff
@@ -386,7 +386,10 @@ UM.MainWindow
                 visible: opacity > 0
                 opacity: base.showPrintMonitor ? 1 : 0
 
-                Behavior on opacity { NumberAnimation { duration: 100; } }
+                // Behavior on opacity { NumberAnimation { duration: 100; } }
```
Comment (nallath) [inline resources/qml/Cura.qml]: Remove commented out code.

File: plugins/UM3NetworkPrinting/PrinterInfoBlock.qml [hunk 3]
```diff
@@ -266,13 +275,18 @@ Rectangle
                                         return "";
                                 }
                             }
+                            if (printer.status == "printing")
+                            {
+                                return catalog.i18nc("@label:status", "Printing")
```
Comment (jackha) [inline plugins/UM3NetworkPrinting/PrinterInfoBlock.qml]: Double checked. Is in catalog.

**Comment by jackha** [comment]
Partly related to this PR, but I get a lot of errors in the log:

`
file:///home/jack/dev/Cura/plugins/UM3NetworkPrinting/PrinterInfoBlock.qml:140:17: QML Image: Cannot anchor to an item that isn't a parent or sibling.
file:///home/jack/dev/Cura/plugins/UM3NetworkPrinting/PrinterInfoBlock.qml:314:33: Unable to assign [undefined] to QUrl
Also in 3.0 branch: file:///home/jack/dev/Cura/plugins/UM3NetworkPrinting/PrinterInfoBlock.qml:357: ReferenceError: showExtended is not defined
file:///home/jack/dev/Cura/plugins/UM3NetworkPrinting/PrinterInfoBlock.qml:140:17: QML Image: Cannot anchor to an item that isn't a parent or sibling.
file:///home/jack/dev/Cura/plugins/UM3NetworkPrinting/PrinterInfoBlock.qml:357: ReferenceError: showExtended is not defined
file:///home/jack/dev/Cura/plugins/UM3NetworkPrinting/PrinterInfoBlock.qml:314:33: Unable to assign [undefined] to QUrl`

File: plugins/UM3NetworkPrinting/PrinterInfoBlock.qml [hunk 4]
```diff
@@ -290,7 +290,7 @@ Rectangle
 
                     Image
                     {
-                        width: 16 * screenScaleFactor
+                        width: statusText.height * screenScaleFactor
```
Comment (fieldOfView) [inline plugins/UM3NetworkPrinting/PrinterInfoBlock.qml]: Remove screenScaleFactor. screenScaleFactor is only used with absolute pixel values. statusText.height already has the devicePixelRatio included.

File: plugins/UM3NetworkPrinting/ClusterMonitorItem.qml [hunk 5]
```diff
@@ -10,8 +10,8 @@ Component
     Rectangle
     {
         width: maximumWidth
-        height: maximumHeight
-        color: "#FFFFFF" // TODO; Should not be hardcoded.
+        height: maximumHeight
```
Comment (fieldOfView) [inline plugins/UM3NetworkPrinting/ClusterMonitorItem.qml]: Remove trailing spaces

File: plugins/UM3NetworkPrinting/PrinterInfoBlock.qml (hunk 4 above)
Comment (jackha) [reply plugins/UM3NetworkPrinting/PrinterInfoBlock.qml]: Hey @fieldOfView hi :-)
Thanks I'll fix it.

File: plugins/UM3NetworkPrinting/ClusterMonitorItem.qml (hunk 5 above)
Comment (jackha) [reply plugins/UM3NetworkPrinting/ClusterMonitorItem.qml]: fixed

**Pull request merged**